### DISCUSSION TITLE
Socials Module

### DISCUSSION THREAD

**Discussion by giscus** [hbstack/site#774] [General]
# Socials Module

The socials module is a helper to generate social links.

https://hbstack.dev/modules/components/socials/

**Comment by pleasantone** [comment]
Docs need updating eg “website”

**Reply by razonyang** [reply]
Thanks for reminding, I've already update locally yesterday, not push to remote yet.

**Comment by LITUATUI** [comment]
Please consider adding [buymeacoffee](https://buymeacoffee.com/). It's a good alternative to Ko-Fi.

**Reply by razonyang** [reply]
Available since [github.com/hbstack/socials@v0.13.0](https://github.com/hbstack/socials/releases/tag/v0.13.0) (`buymeacoffee`).

**Reply by LITUATUI** [reply]
Is there a way to order the social icons? I want all donate platforms to stay together.

**Reply by razonyang** [reply]
Currently it's not possible, unlike `array`, `map` is unordered, that is, it's not ordered as same as the configurations.

Track this on hbstack/header#582, hbstack/footer#284, hbstack/blog#801

**Comment by LITUATUI** [comment]
It would be nice to have [Signal](https://signal.org) too :)

**Reply by razonyang** [reply]
It would be great if you can provide the url example, so I don't need to register or find an account on there.

**Reply by LITUATUI** [reply]
Signal links are pretty long (for security), for example [this one](https://signal.group/#CjQKIF-ShfoXZSAbfcz-GVwWOzmlQsZp-XNuLYYfQD1AasQQEhBxpyLr4Jeb6tf_i4pMOkco) for a group.

Groups: https://signal.group/#xxxxxxxxxxxxxxx
Personal: https://signal.me/#xxxxxxxxxxxxxxxx

Probably it would be best to treat it as a custom URL.

**Reply by razonyang** [reply]
Available since [v0.19.0](https://github.com/hbstack/socials/releases/tag/v0.19.0)

> Probably it would be best to treat it as a custom URL.

As other social media, you can assign any URL as the `id`.
